content/en/continuous_integration/guides/developer_workflows.md

```diff
@@ -22,6 +22,8 @@ CI Test Visibility integrates with other developer-oriented Datadog products as
 - Open tests in GitHub and your IDE
 - [Enabling Test Summaries in GitHub Pull Request Comments][1]
 
+These features are available for all Test Visibility customers, and they do not require usage of the Datadog GitHub App integration.
```

Comment:
I added the change you suggested here, since it actually applies to all of these features! @rtrieu